src/js/message.js

```diff
@@ -28,12 +27,11 @@ class Message {
 
 		this.opts = Object.assign({}, {
 			autoOpen: true,
-			messageClass,
 			type,
-			typeClass: `${messageClass}--${type}`,
+			typeClass: `${this.messageClass}--${type}`,
```

Comment:
There are references in the JS to 'notice-bleed' which I think is redundant? 

Related: I wonder if there's a way to kill `typeNucleus` now bleed is gone? It's one of the more tricky things to understand coming back to this. Thinking something like:

```
const options = {
	type: 'alert',
        inner: true
	...
};
```